### PR TITLE
Changed lots of docs

### LINKED ISSUE #9: Document files and fields for this repo
The tags.json is the tag format for the image.

The psVersions.json is the version of PowerShell we will build an image for.

getLatestTag.ps1  returns a structure telling us a tag to build from and how to represent that tag in the tag template.  Here are the properties of the structure:

| Name | description |
|-------|------------|
| Tag     | What we should tag the image |
| Type   | `Full` or `Short`.  Some images have two representations of the same tag.  A full descriptive tag, and short.  Just say which one the 'Tag' property is.  For example for xenial, a short tag would be `xenial` and a long tag would be `xenial-20180525`.|
| FromTag | the tag we should build the image from.  We should try to use the most descriptive tag.  For example for Ubuntu, I would use `xenial-20180525` |

There is a helper [here](https://github.com/PowerShell/PowerShell-Docker/blob/master/tools/getDockerTags/getDockerTags.psm1) which can generate these for you most of the time.

### LINKED ISSUE #314: Add a doc describing peoples roles
Docker has this doc:
https://github.com/docker/engine/blob/9e7d5ac5ea55dc57185f49476e03d6efa76b3512/MAINTAINERS#L54

Under their terms I would call @RDIL a curator and @kiazhi  `associated people`

cc @RDIL @kiazhi

### PR DESCRIPTION
**Pull request by RDIL**
## PR Summary
* Fixes #314 
* Fixes #9 

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [x] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by RDIL** [comment]
Hey @adityapatwardhan, @kiazhi, and @anmenaga,
do any of you guys have public emails that can be used for the docs?

**Comment by RDIL** [comment]
Got it 👍

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# PowerShell-Docker maintainers file
+# Adapted from Moby's maintainers file
+#
+# This file describes the maintainer groups within the PowerShell/PowerShell-Docker project.
+#
+# It is structured to be consumable by both humans and programs.
+# To extract its contents programmatically, use any TOML-compliant
+# parser.
```
Comment (TravisEz13) [inline MAINTAINERS]: I'm fin with this being plain markdown if  you prefer.  I'm sure docker has a reason for using this format, but I don't

File: docs/development.md [hunk 2]
```diff
@@ -146,3 +119,37 @@ Here is a template for an image build job:
...
+```json
+"tagTemplates": [
+    "#psversion#-windowsservercore-#tag#",
+    "windowsservercore-#tag#"
+]
+```
+
+## Metadata Files
+
+This file *is **required*** for Linux containers. Here is the bare minimum:
```
Comment (TravisEz13) [inline docs/development.md]:
```suggestion
This file *is **required*** for all containers. Here is the bare minimum:
```

File: docs/development.md [hunk 3]
```diff
@@ -146,3 +119,37 @@ Here is a template for an image build job:
...
+
+  * `#psversion#` is replaced by the version of PowerShell used to build the image.
+  * `#tag#` is replaced by all tags generated by the `getLatestTag.ps1` script.
+  * `#shorttag#` is replaced by short tags generated by the `getLatestTag.ps1` script.
+  * `#longtag#` is replaced by long tags generated by the `getLatestTag.ps1` script.
+
+### Example
+
+```json
+"tagTemplates": [
```
Comment (TravisEz13) [inline docs/development.md]: We should move this to be the version in the metadata.

Comment (TravisEz13) [reply docs/development.md]: Can be done later.

File: docs/development.md [hunk 4]
```diff
@@ -146,3 +119,37 @@ Here is a template for an image build job:
...
+Tags are a JSON array that describes the tags the image should have.
+
+### Supported Tags
+
+Tags you can use:
+
+  * `#psversion#` is replaced by the version of PowerShell used to build the image.
+  * `#tag#` is replaced by all tags generated by the `getLatestTag.ps1` script.
+  * `#shorttag#` is replaced by short tags generated by the `getLatestTag.ps1` script.
+  * `#longtag#` is replaced by long tags generated by the `getLatestTag.ps1` script.
```
Comment (TravisEz13) [inline docs/development.md]: longtag is never used, but I'm fine with keeping it.

File: MAINTAINERS [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+    [Org.Alumni]
+
+    # This list contains maintainers that are no longer active, or only slightly
+    # active on the project.
+    # It is thanks to these people that the project has become what it is today.
+    # Thank you!
+
+    people = [
+        "anmenega"
```
Comment (TravisEz13) [inline MAINTAINERS]: He still is a maintainer...

Comment (TravisEz13) [reply MAINTAINERS]: We want to keep at least two.

File: MAINTAINERS (hunk 1 above)
Comment (RDIL) [reply MAINTAINERS]: I like it being TOML compliant.

**Pull request merged**